source/mainnet/smart-contracts/onboarding-guide-solana-developers/overview.rst

```diff
@@ -0,0 +1,112 @@
+.. _solana-overview:
+
+========
+Overview
+========
+
+This overview outlines the main differences of account models and smart contract functionality between Solana and Concordium.
+
+Accounts
+========
+
+Solana accounts are somewhat similar to files and can hold arbitrary data.
+Some accounts are marked as `executable` and contain program code.
+These accounts themselves are immutable, but programs stored in their data can modify data in accounts they own.
+
+Concordium, on the other hand, has a strict separation between user accounts and smart contracts.
+Both accounts and smart contracts have CCD balances associated with them.
+Moreover, smart contracts can have state -- structured data associated with the contract.
+Unlike Solana, this data can be updated directly only by the smart contract code it belongs to.
```

Comment:
How is this `unlike`? (I don't know anything about solana, but the sentence above indicates that state of an "account" can be updated by its associated code)